core-bundle/src/Resources/contao/templates/backend/be_main.html5

```diff
@@ -115,4 +115,4 @@
   <?php endif; ?>
 
 </body>
-</html>
\ No newline at end of file
+</html>
```

Comment:
This seems like an unrelated change that needs to be reverted.

Reply:
done

Reply:
It is still there? 🙈

Reply:
dann check ich ned was er da anmeckert

Reply:
der meckert schon die letzte leere Zeile an oder?

Reply:
No, you have removed the newline at the end of of the file in your PR. You need to revert that (i. e. add the newline again).

Reply:
ich würds ja gern machen. Wenn ich das Ding wieder bearbeiten möchte, ist da schon automatisch ne neue Zeile. Ihr dürft da gerne das einfach ändern.
<img width="509" alt="Bildschirmfoto 2020-04-09 um 10 13 23" src="https://user-images.githubusercontent.com/15326334/78873516-4b23e180-7a4b-11ea-90ae-269f1fab1e0c.png">

Reply:
Ah sorry, it's the other way around. There was no newline before, but now there is ;)

Reply:
die hab ich eigentlich mit den [letzten Änderungen](https://github.com/contao/contao/pull/1415/commits/669f1b30ba0fa654df456b4ca0f03b4be25edda9) entfernt. Nimmts aber irgendwie ned

Reply:
I suppose that happens because you are doing it via GitHub. GitHub automatically adds a newline on save (as this would be standard).

Reply:
Finally. Also doch immer über die IDE :) Danke dir Fritz.